apps/ehr/src/features/external-labs/components/labs-orders/LabsTableRow.tsx

```diff
@@ -94,15 +94,19 @@ export const LabsTableRow = ({
 
   return (
     <TableRow
-      onClick={onRowClick}
       sx={{
         '&:hover': { backgroundColor: '#f5f5f5' },
         cursor: 'pointer',
       }}
     >
-      {columns.map((column) => (
-        <TableCell key={column}>{renderCellContent(column)}</TableCell>
-      ))}
+      <Link
```

Comment:
@VladMstv What do you think, is it worth using Link components that allow opening pages in a new tab? They haven't been used in the project until recently. In principle, there's nothing critical, but opening pages in a new tab can create problems for SPAs. This is not a blocker for the PR, but it would be good to think about whether we continue using Link components.

Reply:
@saewitz I encountered an error here - since the DTO interface changed, and there was undefined in the URL. While fixing this place, I thought it would be better to return to the old version with a click handler, so that logic could be managed from the parent component. But I don't mind if you decide to change it back to a Link.